33023 - H-R-1 Not showing all data

tyndmar opened the pull request:
Commit 1 - ONLY cosmetics, NO logic changes

sewa67 committed:
Ticket - 32176 Prep Listing Report - Does not support negative markup %.
Files - cerep/r-prepN.w

sewa67 committed:
Ticket - 33152 Midwest Container: INK COLOR SET UP - HELP
Files - windows/l-item4.w,windows/l-itmdsc.w

Commit:
33023 - H-R-1 Not showing all data
Commit 1 - ONLY cosmetics, NO logic changes

brad-advantzware reviewed (changes requested):
DEF VAR list-name AS CAHR NO-UNDO.
Also, why are full expanded keywords reduced?  That is not per policy and if you use Progress Dev Studio, is part of the standard beautification that we want as per coding standards.

sewa67 committed:
Ticket - 29549 Customization to reports with RFC logo.
Files - arrep/r-stmt.w, cec/quote/quoxpnt2.i, cec/quote/quoxprnt.p, oe/rep/bolrfcx.i, oe/rep/relacpi,i, oe/rep/relacpi2.i, po/po-xprnt.i,
po/po-xprnt.p

sewa67 committed:
Ticket - 29549 Customization to reports with RFC logo.
Files - arrep/r-stmt.w, oe/rep/bolrfcx.i, oe/rep/relacpi,i, oe/rep/relacpi2.i,
 cec/quote/quorfc2.i, cec/quote/quorfc.p, cec/quote/quorfc.i, sys/ref/sys-ctrl.i, est/r-quoprt.w

sewa67 committed:
Ticket - 33199 Exception Report.
Files - oe/bolcheck.i, oe/bolcheck.p, oerep/r-bolprt.w

Commit:
32839 Improvements to Asi Installer

Commit:
Mods to link FTP and rest of update procedures

ron-advantzware committed:
33348 Fix to handle shift spanning midnight

Commit:
Merge pull request #3096 from Advantzware/feature/33348_Fix_to_handle_shift_spanning_midnight

33348 Fix to handle shift spanning midnight

ron-advantzware committed:
30987 3C TS Job Sort by Start Date

brad-advantzware committed:
33269 D-E-1 discrepancy on a diecut reported on Gopfert

sewa67 committed:
Ticket - 29759 Add Negative Receipt for AWS PO# 103799. Enter Cost of 66,666.67. Save Erases Data.
Files - fg/b-rcptd.w, fg/caclrcptCostfromPo.p

sewa67 committed:
Ticket - 32483 Negative Receipts for Raw Material for item called PRINTING purchased as LOT unit of Measure.
Files - rm/b-rcptd.w

sewa67 committed:
Ticket - 29759 Add Negative Receipt for AWS PO# 103799. Enter Cost of 66,666.67. Save Erases Data.
Files - fg/b-rcptd.w

wade-advantzware committed:
32477 Make sure 2 users cannot select same BOL for posting

Commit:
change testing version

Commit:
32839 - After customer testing - tweaks

Commit:
Merge branch 'develop' into feature/32839_Improvements_to_Asi_Installer

wade-advantzware committed:
32459 Releases Reference Invalid Shiptos

brad-advantzware committed:
33396 FG# creation when creating order

brad-advantzware committed:
33351 Excel import process default file type

brad-advantzware committed:
33351 - Code formatting only

Commit:
32839 - tweaks and updates following customer installs

Commit:
Merge branch 'feature/32839_Improvements_to_Asi_Installer' of https://github.com/Advantzware/Advantzware into feature/32839_Improvements_to_Asi_Installer

Commit:
Merge pull request #3100 from Advantzware/feature/32839_Improvements_to_Asi_Installer

32839 - After customer testing - tweaks

sewa67 committed:
Ticket- 32856 BOL Overlap
Files - oe/rep/bolprem.i,oe/rep/bolprem23.i,oe/rep/bolprem3.i

sewa67 committed:
Ticket - 32456 Lot (L) versus M (Thousand) price extension in O-R-11 report.
Files - oe/priceProcs.p

Commit:
Merge pull request #3073 from Advantzware/feature/33152_Midwest_Container_INK_COLOR_SET_UP_HELP

Ticket - 33152 Midwest Container: INK COLOR SET UP - HELP

Commit:
32839 - fix to support old advantzware.usr file format (16.7.5 and earlier)

Commit:
Merge pull request #3110 from Advantzware/feature/32839_Improvements_to_Asi_Installer

32839 - fix to support old advantzware.usr file format (16.7.5 and ea…

Commit:
33023 - H-R-1 Not pulling required data

brad-advantzware committed:
32861 Errors when entering order from estimate with purchased items

sewa67 committed:
Ticket - 33477 Fright BOL incorrect - Unit values are all defaulting to 1, they need to match the BOL output
Files - oe/rep/bolfiftn.p,oe/rep/bolfiftn2.i,oe/rep/bolfrftn.p,oe/rep/bolfrftn2.i

sewa67 committed:
Ticket - 33466 POPRINT 1 vs PRPRINT 10.
Files - po/po-xprnt.p, po/po-xprt2.p, po/po-xprt20.p, po/po-xprnt10.p

sewa67 committed:
Ticket - 33479 Error message when editing the set header on a partition.
Files - cec/d-updsetpart.w

sewa67 committed:
Ticket - 33478 Date Range issue and error in editing that sticks user in a loop.
Files - ap/v-apinv.w

brad-advantzware committed:
33597 AU1 Import for Lanco York

sewa67 committed:
Ticket - 33479 Error message when editing the set header on a partition.
Files - cec/d-updsetpart.w

sewa67 committed:
Ticket - 32483 Negative Receipts for Raw Material for item called PRINTING purchased as LOT unit of Measure.
Files - rm/b-rcptd.w

sewa67 committed:
Ticket - 29759 Add Negative Receipt for AWS PO# 103799. Enter Cost of 66,666.67. Save Erases Data
Files - fg/b-rcptd.w

sewa67 committed:
Ticket - 30703 Receiving Set Header Components with no Job/PO
Files - fg/b-rcptd.w,fg/cmpQtyMsg.w

sewa67 committed:
Ticket - 32605 Cust Code and Cust Name in same column in excel
Files - ar/ar-agingN.i,arrep/r-aragedN.w

sewa67 committed:
Ticket -32480 ER5 Report : Orders without Estimates
 Files - cerep/r-estN.w

Commit:
Merge pull request #3115 from Advantzware/feature/33478_Date_Range_issue_and_error_in_editing_that_sticks_user_in_a_loop

Ticket - 33478 Date Range issue and error in editing that sticks user in a loop.

Commit:
Merge pull request #3112 from Advantzware/feature/33477_Fright_BOL_incorrect_Unit_values_are_all_defaulting_to_1_they_need_to_match_the_BOL_output

Ticket - 33477 Fright BOL incorrect - Unit values are all defaulting to 1, they need to match the BOL output

Commit:
Merge pull request #3113 from Advantzware/feature/33466_POPRINT1_vs_PRPRINT10

Ticket - 33466 POPRINT 1 vs PRPRINT 10.

Commit:
Merge pull request #3090 from Advantzware/feature/33199_Exception_Report

Ticket - 33199 Exception Report.

Commit:
Merge pull request #3107 from Advantzware/feature/32856_BOL_Overlap

Ticket- 32856 BOL Overlap

Commit:
Merge pull request #3103 from Advantzware/feature/32459_Releases_reference_invalid_shiptos

32459 Releases Reference Invalid Shiptos

Commit:
Merge pull request #3108 from Advantzware/feature/32456_Lot__L_versus_M__Thousand_price_extension_in_OR11_report

Ticket - 32456 Lot (L) versus M (Thousand) price extension in O-R-11 report.

Commit:
Merge pull request #3097 from Advantzware/feature/30987_3C_TS_Job_Sort_by_Start_Date

30987 3C TS Job Sort by Start Date

Commit:
Merge pull request #3104 from Advantzware/feature/33396_FG#_creation_when_creating_order

33396 FG# creation when creating order

Commit:
Merge pull request #3105 from Advantzware/feature/33351_Excel_import_process_default_file_type

Feature/33351 excel import process default file type

Commit:
Merge pull request #3102 from Advantzware/feature/33269_D-E-1_discrepancy_on_a_diecut_reported_on_Gopfert

33269 D-E-1 discrepancy on a diecut reported on Gopfert

Commit:
Merge pull request #3111 from Advantzware/feature/32861_Errors_when_entering_order_from_estimate_with_purchased_items

32861 Errors when entering order from estimate with purchased items

Commit:
Merge pull request #3114 from Advantzware/feature/33479_Error_message_when_editing_the_set_header_on_a_partition

Ticket - 33479 Error message when editing the set header on a partition.

Commit:
Merge pull request #3121 from Advantzware/feature/32480_ER5_Report_Orders_without_Estimates

Ticket -32480 ER5 Report : Orders without Estimates

Commit:
Merge pull request #3120 from Advantzware/feature/32605_Cust_Code_and_Cust_Name_in_same_column_in_excel

Ticket - 32605 Cust Code and Cust Name in same column in excel

sewa67 committed:
Ticket - 32483 Negative Receipts for Raw Material for item called PRINTING purchased as LOT unit of Measure
Files - rm/b-rcptd.w

sewa67 committed:
Ticket - 29759 Add Negative Receipt for AWS PO# 103799. Enter Cost of 66,666.67. Save Erases Data
Files - fg/b-rcptd.w,write.trg/fg-rctd.p

sewa67 committed:
Ticket - 29759 Add Negative Receipt for AWS PO# 103799. Enter Cost of 66,666.67. Save Erases Data
Files - fg/b-rcptd.w

sewa67 committed:
Ticket - 30703 Receiving Set Header Components with no Job/PO
Files - fg/b-rcptd.w

sewa67 committed:
Ticket - 32400 Modification Request to IR4 Report - Add CSR Column for display/search.
Files - fgrep/r-qtyvlN.i, fgrep/r-qtyvlN.w

sewa67 committed:
Ticket - 32851 Convert prep/routing tab to a pop up box for entry and editing purposes in Corrugated/Folding
Files - ce/b-estop.w,est/b-estop.w,est/b-estprp.w,est/w-est.w,est/w-estc.w,windows/l-prep.w,ce/d-estop.w,
              est/d-estop.w,est/d-estprp.w,est/vp-estop.w,est/vp-estprp.w,windows/l-codedscr.w

sewa67 committed:
Ticket - 29765 Cannot see Update and other buttons on E-B-5 screen
Files - browsers/st-pattn.w

sewa67 committed:
Ticket - 28407 Post to Raw materials file - new login to prevent posting invalid Tag #
Files - rm/r-rmte&p.w

sewa67 committed:
Ticket - 32041 <New>NK1-QuoPrint=Chattanooga - new quote format.
Files - est/r-quoprt.w, sys/ref/sys-ctrl.i, cec/quote/quochatt.i, cec/quote/quochatt.p, cec/quote/quochatt10.i,

Commit:
Merge pull request #3099 from Advantzware/feature/32483_Negative_Receipts_for_Raw_Material_for_item_called_PRINTING_purchased_as_LOT_unit_of_Measure

Ticket - 32483 Negative Receipts for Raw Material for item called PRINTING purchased as LOT unit of Measure.

Commit:
Merge pull request #3122 from Advantzware/feature/32400_Modification_Request_to_IR4_Report__Add_CSR_Column_for_display_search

Ticket - 32400 Modification Request to IR4 Report - Add CSR Column for display/search.

Commit:
Merge pull request #3039 from Advantzware/feature/32176_Prep_Listing_Report_Does_not_support_negative_markup

Ticket - 32176 Prep Listing Report - Does not support negative markup %.

Commit:
Merge pull request #3119 from Advantzware/feature/30703_Receiving_Set_Header_Components_with_no_Job_PO

Ticket - 30703 Receiving Set Header Components with no Job/PO

Commit:
Merge branch 'develop' into feature/29759_Add_Negative_Receipt_for_AWS_PO_103799_Enter_Cost_of_6666667_Save_Erases_Data

Commit:
Merge pull request #3098 from Advantzware/feature/29759_Add_Negative_Receipt_for_AWS_PO_103799_Enter_Cost_of_6666667_Save_Erases_Data

Ticket - 29759 Add Negative Receipt for AWS PO# 103799. Enter Cost of  66,666.67. Save Erases Data.

ron-advantzware committed:
33701 16.7.16 Upgrade - Touch Screen midnight cross over was not fixed

Commit:
33477 - NK1-BolFmtX - case count correction per Clay

Commit:
Merge pull request #3129 from Advantzware/feature/33477_-_NK1-BolFmtx

33477 - NK1-BolFmtX - case count correction per Clay

Commit:
33477 - additional mods

Commit:
Merge pull request #3130 from Advantzware/feature/33477_-_NK1-BolFmtx

33477 - additional mods

ron-advantzware committed:
32152 New NK1/Settings User Interface

Commit:
Merge pull request #3128 from Advantzware/feature/33701_16.7.16_Upgrade_-_Touch_Screen_midnight_cross_over_was_not_fixed

33701 16.7.16 Upgrade - Touch Screen midnight cross over was not fixed

sewa67 committed:
Ticket - 29549 Customization to reports with RFC logo
Files - oe/rep/bolrfcx.i, oe/rep/bolrfcx2.i, oe/rep/bolrfcx3.i, oe/rep/bolrfcxcons.i, oe/rep/bolrfcxcons2.i, cec/quote/quorfc.p, cec/quote/quorfc2.i,   cec/quote/quorfc.i,  oerep/r-relprt.w, oe/rep/relrfc.p, oe/rep/relrfc.i, oe/rep/relrfc2.i,arrep/r-stmt.w,sys/ref/sys-ctrl.i

sewa67 committed:
Ticket - 32605 Cust Code and Cust Name in same column in excel
Files - ar/ar-agingN.i,arrep/r-aragedN.w

sewa67 committed:
Update r-rmte&p.w

Message correction

sewa67 committed:
Ticket - 28343 GL account in TS Employee Machine tab - Not used - change to Note.
Files - addon/browsers/employee.w, addon/methods/lstlogic/custom/employe_.i,, addon/viewers/employee.w,
addon/windows/emp-expe.w, addon/windows/employee.w

sewa67 committed:
Ticket - 33478 Date Range issue and error in editing that sticks user in a loop.
Files - ap/v-apinv.w

sewa67 committed:
Ticket - 32851 Convert prep/routing tab to a pop up box for entry and editing purposes in Corrugated/Folding
Files - Files - ce/b-estop.w,est/b-estop.w,est/b-estprp.w,est/w-est.w,est/w-estc.w,windows/l-prep.w,ce/d-estop.w,
            est/d-estop.w,est/d-estprp.w,windows/l-codedscr.w,p-estop,panel/p-estprp.w,est/w-estcf.w,est/w-estesf.w,
            est/w-estf.w,est/w-invesf.w,est/w-invest.w,est/w-jobesf.w,est/w-jobest.w,est/w-ordesf.w,est/w-ordest.w

Commit:
Merge pull request #3101 from Advantzware/feature/32477_Make_sure_2_users_cannot_select_same_BOL_for_posting

32477 Make sure 2 users cannot select same BOL for posting

Commit:
Merge pull request #3127 from Advantzware/feature/32041_New_NK1_QuoPrint_Chattanooga_new_quote_format

Ticket - 32041 <New>NK1-QuoPrint=Chattanooga - new quote format.

Commit:
Merge pull request #3125 from Advantzware/feature/29765_Cannot_see_Update_and_other_buttons_on_EB5_screen

Ticket - 29765 Cannot see Update and other buttons on E-B-5 screen

Commit:
Merge pull request #3132 from Advantzware/feature/32152_New_NK1/Settings_User_Interface

32152 New NK1/Settings User Interface

ron-advantzware committed:
32152 New NK1 v2

Commit:
Merge pull request #3136 from Advantzware/feature/32152_New_NK1_v2

32152 New NK1 v2

ron-advantzware committed:
32152 New NK1 v3

sewa67 committed:
Ticket - 33753 N-K BOLFMT = SOULEMED (Total Units is incorrect when printing multiple bill of ladings in a batch).
Files - oe/rep/bolsolmed.p, oe/rep/bolacpi.p, oe/rep/bolchict.p, oe/rep/bolcscga.p, oe/rep/bolelite.p, oe/rep/bolfiftn.p, oe/rep/bolfrftn.p, oe/rep/bolkdwlx.p,oe/rep/bolloyln.p, oe/rep/bollprint.p, oe/rep/bolmich.p, oe/rep/bolnstok.p, oe/rep/bolprem.i, oe/rep/bolprmbr.i, oe/rep/bolrfcx.i, oe/rep/boltrilx.p,oe/rep/bolxcorc.p, oe/rep/bolxinld.p, oe/rep/bolmidx.p

Commit:
Merge pull request #3137 from Advantzware/feature/32152_New_NK1_v3

32152 New NK1 v3

Commit:
Merge pull request #3135 from Advantzware/feature/33478_Date_Range_issue_and_error_in_editing_that_sticks_user_in_a_loop

Ticket - 33478 Date Range issue and error in editing that sticks user in a loop.

Commit:
Merge pull request #3133 from Advantzware/feature/32605_Cust_Code_and_Cust_Name_in_same_column_in_excel

Ticket - 32605 Cust Code and Cust Name in same column in excel

Commit:
Merge branch 'develop' into feature/29549_Customization_to_reports_with_RFC_logo

Commit:
Merge pull request #3088 from Advantzware/feature/29549_Customization_to_reports_with_RFC_logo

Ticket - 29549 Customization to reports with RFC logo.

Commit:
Merge pull request #3126 from Advantzware/feature/28407_recorrect_Post_to_Raw_materials_file_new_login_to_prevent_posting_invalidTag

Ticket - 28407 Post to Raw materials file - new login to prevent posting invalid Tag #

brad-advantzware committed:
33762 Estimate Print Tab - Calculate Button - Check Toggle Box to Recalc Run Speeds is not Importing Machine File Speeds

brad-advantzware committed:
33240 Missing feature - Price Hold is to put Order on Hold

brad-advantzware committed:
33240 Missing feature - Price Hold is to put Order on Hold

wade-advantzware committed:
33949 Error while posting FG

sewa67 committed:
Ticket - 34083 Large costs prevent viewing invoice in OQ1
Files oeinq/b-oeinvl.w

sewa67 committed:
Ticket - 34084 2 Page BOLfmt20 - Page number issue and logo printing issue
Files - oe/rep/bolxprt30.i,oe/rep/bolxpr20con.i

sewa67 committed:
Ticket - 33998 Character Limit on Cost For Prep/Die File.
Files - viewers/prep.w

sewa67 committed:
Ticket - 33830 IF1 Import Question for FG Items requiring a customer code.
Files - viewers/itemfg.w

Commit:
Merge pull request #3140 from Advantzware/feature/33762_EC_-_Tandem_Estimates_-_Check_Toggle_Box_to_Recalc_Run_Speeds_is_not_Importing_Machine_File_Speeds

33762 Estimate Print Tab - Calculate Button - Check Toggle Box to Rec…

Commit:
Merge pull request #3138 from Advantzware/feature/33753_N_K_BOLFMT_SOULEMED_Total_Units_is_incorrect_when_printing_multiple_bill_of_ladings_in_a_batch

Ticket - 33753 N-K BOLFMT = SOULEMED (Total Units is incorrect when printing multiple bill of ladings in a batch).

Commit:
Merge pull request #3123 from Advantzware/feature/32851_Convert_prep_routing_tab_to_a_pop_up_box_for_entry_and_editing_purposes_in_Corrugated_Folding

Ticket - 32851 Convert prep/routing tab to a pop up box for entry and editing purposes in Corrugated/Folding

Commit:
Merge pull request #3134 from Advantzware/feature/28343_GL_account_in_TS_Employee_Machine_tab_Not_used_change_to_Note

Ticket - 28343 GL account in TS Employee Machine tab - Not used - change to Note.

sewa67 committed:
Ticket - 33463 Request - Adding UOM to IC1
Files - fgrep/r-cycntN.w

sewa67 committed:
Ticket - 34143 Expand Selling Price Character Limit in Quotes.
Files - est/b-qtqty.w

sewa67 committed:
Ticket - 33830 IF1 Import Question for FG Items requiring a customer code.
Files - viewers/itemfg.w

Commit:
Update b-qtqty.w

Commit:
Merge pull request #3147 from Advantzware/feature/34143_Expand_Selling_Price_Character_Limit_in_Quotes

Ticket - 34143 Expand Selling Price Character Limit in Quotes.

Commit:
Merge pull request #3142 from Advantzware/feature/34083_Large_costs_prevent_viewing_invoice_in_OQ1

Ticket - 34083 Large costs prevent viewing invoice in OQ1

Commit:
Merge pull request #3144 from Advantzware/feature/33998_Character_Limit_on_Cost_For_Prep_Die_File

Ticket - 33998 Character Limit on Cost For Prep/Die File.

Commit:
Update printBOL.p

Commit:
Merge pull request #3141 from Advantzware/feature/33240_Missing_feature_-_Price_Hold_is_to_put_Order_on_Hold

33240 Missing feature - Price Hold is to put Order on Hold

sewa67 committed:
Ticket - 33999 Color Carton - H-C
Files - sa/sa-mtdsl.i

ron-advantzware committed:
32152 New NK1 v4

sewa67 committed:
Ticket - 34143 Expand Selling Price Character Limit in Quotes.
Files - est/b-qtest.w, est/b-qtitm.w, est/b-qtqty.w

sewa67 committed:
Ticket - 33998 Character Limit on Cost For Prep/Die File.
files - cec/box/pr42-prp.p, cec/box/pr42mis2.p, cec/pr4-prp.p, cec/pr4-mis2.p

sewa67 committed:
Ticket - 28567 Change to O-R-5
Files -oerep/r-bookedN.i,oerep/r-bookedN.w

sewa67 committed:
Ticket - 33830 IF1 Import Question for FG Items requiring a customer code
Files - viewers/itemfg.w

Commit:
Merge pull request #3148 from Advantzware/feature/33999_Color_Carton_H_C

Ticket - 33999 Color Carton - H-C

Commit:
Merge pull request #3146 from Advantzware/feature/33463_Request_Adding_UOM_to_IC1

Ticket - 33463 Request - Adding UOM to IC1

Commit:
Merge pull request #3118 from Advantzware/feature/33597_AU1_Import_for_Lanco_York

33597 AU1 Import for Lanco York

Commit:
compile issues

wade-advantzware committed:
33949 Error while posting FG

brad-advantzware committed:
30964 - Cosmetic Changes Only

brad-advantzware committed:
30964 Job Farm Tab Does not Pull Correct Cost From Estimate

sewa67 committed:
Ticket - 28567 Change to O-R-5
Files - ao/BL/r-booked.i,aoa,tempTable/ttOrdersBooked.i,oerep/r-bookedN.i,oerep/r-bookedN.w

sewa67 committed:
Ticket - 33830 IF1 Import Question for FG Items requiring a customer code.
Files - viewers/itemfg.w, write.trg/itemfg.p

sewa67 committed:
Ticket  - 34202 NK1-JobCardF=Colonial - Add RM Item # to Factory Ticket
Files - cerep/jobcolnl.p

sewa67 committed:
Ticket - 31171 Folding Estimate - Remove margin % from Print tab.
Files - ce/probe.w

sewa67 committed:
Ticket - 34201 Recent update broke the OZ4 report
Files - oerep/r-shed3N.w

Commit:
Merge pull request #3156 from Advantzware/feature/30964_Job_Farm_Tab_Does_not_Pull_Correct_Cost_From_Estimate

Feature/30964 job farm tab does not pull correct cost from estimate

Commit:
Merge branch 'develop' into feature/34143_Correct_Expand_Selling_Price_Character_Limit_in_Quotes

Commit:
Merge pull request #3150 from Advantzware/feature/34143_Correct_Expand_Selling_Price_Character_Limit_in_Quotes

Ticket - 34143 Expand Selling Price Character Limit in Quotes.

Commit:
Merge pull request #3151 from Advantzware/feature/33998_Character_Limit_on_Cost_For_Prep_Die_File

Ticket - 33998 Character Limit on Cost For Prep/Die File.

Commit:
Merge branch 'develop' into feature/33949_Error_while_posting_FG

Commit:
Merge pull request #3153 from Advantzware/feature/33949_Error_while_posting_FG

33949 Error while posting FG

Commit:
Merge pull request #3149 from Advantzware/feature/32152_New_NK1_v4

32152 New NK1 v4

Commit:
Merge pull request #3152 from Advantzware/feature/28567_Change_to_OR5

Ticket - 28567 Change to O-R-5

brad-advantzware committed:
26492 - HM1 Total Projected template calculation fixed

brad-advantzware committed:
33597 ImportAR - type problem with fields

Commit:
Merge pull request #3157 from Advantzware/feature/34202_NK1_JobCardF_Colonial_Add_RM_Item_to_Factory_Ticket

Ticket  - 34202 NK1-JobCardF=Colonial - Add RM Item # to Factory Ticket

Commit:
Merge pull request #3160 from Advantzware/feature/34201_Recent_update_broke_the_OZ4_report

Ticket - 34201 Recent update broke the OZ4 report

Commit:
Merge pull request #3158 from Advantzware/feature/31171_Folding_Estimate_Remove_margin_from_Print_tab

Ticket - 31171 Folding Estimate - Remove margin % from Print tab.

Commit:
Merge pull request #3163 from Advantzware/feature/33597_AU1_Import_for_Lanco_York

33597 ImportAR - type problem with fields

Commit:
Merge pull request #3162 from Advantzware/feature/26492_HM1_-_Profit_Projection_-_FIx_Total_Projected_$PerMSF

26492 - HM1 Total Projected template calculation fixed

Commit:
minor compile error fixes

brad-advantzware committed:
33417 Import Form from Excel - Salesrep not defaulting properly

sewa67 committed:
Ticket - 33830 IF1 Import Question for FG Items requiring a customer code
Files - create.trg/itemfg.p,viewers/itemfg.w

sewa67 committed:
Ticket - 34084 2 Page BOLfmt20 - Page number issue and logo printing issue
Files - oe/rep/bolxpr23.i

sewa67 committed:
Ticket - 34285 SHT CALC Button Bug shows No Lock. This.
Files - jc/shtcalc.p

sewa67 committed:
Ticket - 29549 Customization to reports with RFC logo.
Files - arrep/r-stmt.w, cec/quote/quorfc2.i, oe/rep/bolrfcx.1

brad-advantzware committed:
30356 Error Message when Posting FG

brad-advantzware committed:
32415 - PO Receipt Processing - update of standard IF1 costs

sewa67 committed:
Ticket - 34084 2 Page BOLfmt20 - Page number issue and logo printing issue
Files - oe/rep/bolxpr23.i

Commit:
Merge pull request #3170 from Advantzware/feature/30356_Error_Message_when_Posting_FG

30356 Error Message when Posting FG

Commit:
Merge pull request #3167 from Advantzware/feature/34285_SHT_CALC_Button_Bug_shows_No_Lock_This

Ticket - 34285 SHT CALC Button Bug shows No Lock. This.

Commit:
Merge pull request #3143 from Advantzware/feature/34084_2_Page_BOLfmt20_Page_number_issue_and_logo_printing_issue

Ticket - 34084 2 Page BOLfmt20 - Page number issue and logo printing issue

Commit:
Merge pull request #3145 from Advantzware/feature/33830_IF1_Import_Question_for_FG_Items_requiring_a_customer_code

Ticket - 33830 IF1 Import Question for FG Items requiring a customer code.

Commit:
Merge pull request #3168 from Advantzware/feature/29549_Recorrected_Customization_to_reports_with_RFC_logo

Ticket - 29549 Customization to reports with RFC logo.

Commit:
Merge pull request #3165 from Advantzware/feature/33417_Import_Form_from_Excel_-_Salesrep_not_defaulting_properly

33417 Import Form from Excel - Salesrep not defaulting properly

Commit:
Merge pull request #3171 from Advantzware/feature/32415_O-U-7_LoadTags_Did_Not_Update_I-F-1_View_Item_Cost

32415 - PO Receipt Processing - update of standard IF1 costs

brad-advantzware committed:
33023 H-R-1 Not showing all data - revert changes from #25753

brad-advantzware committed:
Merge branch 'feature/33023_H-R-1_Not_showing_all_data' of https://github.com/Advantzware/Advantzware into feature/33023_H-R-1_Not_showing_all_data
